klog.go

```diff
@@ -1586,3 +1586,87 @@ type LogFilter interface {
 func SetLogFilter(filter LogFilter) {
 	logging.filter = filter
 }
+
+type TraceSimple struct {
```

Comment:
Let's not export types unless it's needed for the API.

Reply:
Done.